Update 01-connecting.md (#34)
hpc-carpentry/hpc-shell: _episodes/01-connecting.md

> to be installed. There are a variety of programs that can be used for this; two common ones we

Reviewer: `two` -> `a few`

Author: ok

describe here, as follows:

#### Git BASH

Git BASH gives you a terminal like interface in Windows. You can use this to connect to a remote 

> computer by SSH. It is can be downloaded for free from

Reviewer 2: ```suggestion
computer with `ssh`. It can be downloaded for free from
```

Reviewer 2: Would love a native speaker's comment on this, @tkphd @ocaisa @reid-a!

Author: SSH rather than `ssh` seems to be used in this lesson - am ok with either, but it would be good to be consistent.

Author: is should be removed

Reviewer: I agree that SSH is used elsewhere. I usually say `via SSH`, but I think that might be nitpicking.

Author: Removed "is", kept "SSH", used "via SSH" 

Thanks for the suggestions

[here](https://gitforwindows.org/).

#### Windows Subsystem for Linux

> The Windows Subsystem for Linux also allows you to connect to a remote computer by using SSH.

Reviewer: Now that I started nitpicking :) , I would use the same phrasing here as above: `connect to a remote computer via SSH`

Author: Made change.

> Instructions on installing it can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10)

Reviewer 2: ```suggestion
Instructions on installing it can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
```
